bundle/jsky.app.ot/src/main/java/jsky/app/ot/progadmin/AdminDialog.java

```diff
@@ -44,8 +37,9 @@ ISPProgram show(IDBDatabaseService database, ISPProgram prog)  {
 
         pane.setMessage(ui);
 
-        JDialog dialog = pane.createDialog(owner, title);
+        JDialog dialog = pane.createDialog(null, title);
```

Comment:
Why is the `owner` suddenly unimportant? Do we want an OS-wide modal dialog, or does this have some other behaviour?

Indeed, according to the Java API docs, it doesn't seem that the use of `null` here is even clearly stated. Can we always assume it will work properly?

If we are going to go this route, it seems that using `pane.createDialog(title)` might the correct method call to make?